package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "publish",
+  "version": "0.1.0-dev",
```

Comment:
unless you think the committed version number should not include the pre-release version in it, which could also make sense.